internal/e2e/log/main.go

```diff
@@ -2,10 +2,15 @@
 
 package main
 
-import httpwasm "github.com/http-wasm/http-wasm-guest-tinygo"
+import httpwasm "github.com/http-wasm/http-wasm-guest-tinygo/handler"
+
+func logAround() {
+	httpwasm.Log("before")
+	defer httpwasm.Log("after")
+
+	httpwasm.Next()
+}
 
 func main() {
-	httpwasm.Log("msg")
-	httpwasm.Log("msg1")
-	httpwasm.Log("msg")
+	httpwasm.HandleFn = logAround
```

Comment:
@anuraaga PS I'm not really sure what's the cleanest way to overwrite an entrypoint function. ideas welcome!